docs/configuration.md

```diff
@@ -698,39 +698,6 @@ This storage backend uses Amazon's Simple Storage Service (S3).
   </tr>
 </table>
 
-### Maintenance
```

Comment:
exactly, it's not part of a particular storage driver. I wanted to move the storage drivers so that they were together, and the maintenance was below them, as I thought was originally intended. I can move it to the top instead if you prefer that.